docs/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import sys
+from pathlib import Path
+
+# Path for package
+sys.path.insert(0, Path(__file__).parents[2].resolve().as_posix())
+# Path for custom extension
+sys.path.insert(0, str(Path(__file__).parent.absolute()))
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "UCB Prefect Tools"
+copyright = "2024, OIT Data Services"
+author = "OIT Data Services"
+release = "v6.2.1"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "task_autodoc",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.viewcode",
```

Comment:
```suggestion
    "sphinx.ext.viewcode",
    "sphinx_autodoc_typehints",
```

Not sure if you've come across this extension before, but being able to leverage Python's built-in type hints would be nice. You mention in the description that you're saving the annotation of individual functions for a subsequent PR, so I wanted to mention this earlier rather than later. Here's [a link](https://github.com/tox-dev/sphinx-autodoc-typehints) to the `autodoc_typehints` GH repository. We'd also need to add the following to the appropriate `requirements.txt`:

```
sphinx-autodoc-typehints==2.2.3
```